CHANGELOG.md

```diff
@@ -49,6 +53,57 @@ aws lambda invoke --function-name $PREFIX-data-migration1 \
 - (Optional) Update the `task-config` for all workflows that use the `sync-granule` task to include `workflowStartTime` set to
 `{$.cumulus_meta.workflow_start_time}`. See [here](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/sync_granule_workflow.asl.json#L9) for an example.
 
+##### After the `cumulus` deployment
+
+As part of the work on the RDS Phase 2 feature, it was decided to re-add the
+granule file `type` property on the file table (detailed reasoning
+https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=219186829).  This
+change was implemented as part of CUMULUS-2672/CUMULUS-2673, however granule
+records ingested prior to v11 *not* have the file.type property stored in the
```

Comment:
```suggestion
records ingested prior to v11 will *not* have the file.type property stored in the
```
```suggestion
records ingested prior to v11 *not* have the file.type property stored in the
```

Reply:
I *think* I took the commit suggestion, but GH is being odd.